tests/modules/mhct-lookup.js

```diff
@@ -77,7 +78,20 @@ test('getFilter', suite => {
         ));
         sinon.reset();
 
-    });    suite.test('Module Cleanup', t => {
+    });  suite.test('given input that can\'t be turned into a truthy string - returns undefined', t => {
+        const inputs = [
+            '',
+            undefined,
+        ];
+        t.plan(inputs.length);
+        inputs.forEach(input => t.deepEqual(
+            getConvertibles(input),
```

Comment:
I think that tests for `getConvertibles` should be in a different suite than the `getFilter` suite. (This applies for the above tests, too, but don't modify those in this PR -- we will fix their suite "membership" later).

Reply:
Got it! I went and moved it into its own suite and added another test.

Reply:
It looks like you copied it to its own suite, rather than moved it.